app/scripts/views/complete_sign_up.js

```diff
@@ -60,6 +64,17 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       return self.fxaClient.verifyCode(uid, code)
           .then(function () {
             self.logScreenEvent('verification.success');
+            var account = self.getAccount();
+
+            if (account.get('needsOptedInToMarketingEmail')) {
```

Comment:
We could attempt to opt-in from the original confirmation tab also, as a stop gap. But I wonder how many people verify in a different browser? I didn't see any explicit metrics being collected around that– but it would be useful to know.